matlab/matlab-client/src/matlab/PAsolve.m

```diff
@@ -682,7 +703,7 @@ function initSolveConfig(solve_config,opt,sched)
             if ismember(c, stdclasses) || iscom(argi{k}) || isjava(argi{k}) || isinterface(argi{k})
             else
                 if ~isfield(funcDatabase, c) || ~isfield(funcDatabase.(c),'dep')
-                    [parammfiles paramclassdirs] = sched.findDependency(c);
+                    [parammfiles paramclassdirs] = sched.findDependency([], c);
```

Comment:
Maybe a new function `findDependency` that takes only the class name could be defined so that you can use the `findDependency` function as it was before changing the signature?